pulsar-client-admin-api/pom.xml

```diff
@@ -35,20 +35,54 @@
     <dependencies>
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>pulsar-common</artifactId>
+            <artifactId>pulsar-client-api</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
 
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>pulsar-client-original</artifactId>
-            <version>${project.parent.version}</version>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <scope>provided</scope>
```

Comment:
This API package shouldn't depend on anything more than `pulsar-client-api`. 

Having the dependencies marked as "provided" doesn't really fix the issue because the implementation will still be using them. We should instead make sure that the POJOs are not using any other dependency.